com.unity.render-pipelines.high-definition/Documentation~/Post-Processing-Depth-of-Field.md

```diff
@@ -45,7 +45,7 @@ Depth Of Field includes [more options](More-Options.md) that you must manually e
 | -------------------------- | ------------------------------------------------------------ |
 | **Resolution**             | Use the drop-down to set the resolution at which HDRP processes the depth of field effect. If you target consoles that use a very high resolution (for example, 4k), select **Quarter,** because it is less resource intensive.<br />&#8226; **Quarter**: Uses quarter the screen resolution.<br />&#8226; **Half**: Uses half the screen resolution.<br />This property only appears when you enable [more options](More-Options.md). |
 | **High Quality Filtering** | Enable the checkbox to make HDRP use bicubic filtering instead of bilinear filtering. This increases the resource intensity of the Depth Of Field effect, but results in smoother visuals.<br />This property only appears when you enable [more options](More-Options.md). |
-| **Physically Based** | Enable the checkbox to make HDRP use a more accurate but slower physically-based technique for the computation of Deph-of-Field. It is highly recommended to enable [Temporal anti-aliasing (TAA)](Anti-Aliasing) at the same time, for improved quality and performance.|
+| **Physically Based** | Enable the checkbox to make HDRP use a more accurate but slower physically-based technique for the computation of Deph-of-Field. It is highly recommended to enable [Temporal anti-aliasing (TAA)](Anti-Aliasing) at the same time, for improved quality especially when using a low number of samples. <br /> The amount of defocus blur will be different when this option is enabled and disabled. When enabled, the defocus blur will be closer to what you would expect from a real-world camera with a configuration matching the one in the physical camera parameters in [HD Camera](HDRP-Camera). However, the maximum radius of the defocus blur is still capped by (a multiple of) the **Max Radius** parameter for performance and quality reasons. |
```

Comment:
The amount of defocus blur differs depending on this value. When enabled, the defocus blur is closer to what you would expect from a real-world camera with a configuration that matches the [Camera's](HDRP-Camera.md) physical camera properties. However, it is not exactly the same as a real-world camera because HDRP caps the maximum radius of the defocus blur (using the **Max Radius** property) for performance and quality reasons.

Reply:
Done!